<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RGGEditPinParameters</class>
 <widget class="QWidget" name="RGGEditPinParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PinCell Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>392</width>
        <height>594</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="0" colspan="3">
        <widget class="QTableWidget" name="piecesTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>355</width>
           <height>120</height>
          </size>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Pieces:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="addLayerBeforeButton">
           <property name="text">
            <string>Add Before</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addLayerAfterButton">
           <property name="text">
            <string>Add After</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="deleteLayerButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Z Origin:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="addPieceButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="deletePieceButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QTableWidget" name="layersTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QDoubleSpinBox" name="Z0rigin">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-10000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="CellMaterial"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Label:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="labelLineEdit">
         <property name="text">
          <string>nuclear pin</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="nameLineEdit">
         <property name="text">
          <string>pin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Cell Material:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="3" column="1">
          <widget class="QCheckBox" name="isHexCheckbox">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, CMB would create a hexagon nuclear pin. Otherwise it would be a rectilinear nuclear pin.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Hex</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="cutAwayViewCheckBox">
           <property name="text">
            <string>Cut-away View</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="infoButton">
           <property name="maximumSize">
            <size>
             <width>238</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Info</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="3">
          <widget class="QPushButton" name="applyButton">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>labelLineEdit</tabstop>
  <tabstop>piecesTable</tabstop>
  <tabstop>addPieceButton</tabstop>
  <tabstop>deletePieceButton</tabstop>
  <tabstop>layersTable</tabstop>
  <tabstop>addLayerBeforeButton</tabstop>
  <tabstop>addLayerAfterButton</tabstop>
  <tabstop>deleteLayerButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
